modules/lvms-creating-lvmcluster-using-rhacm.adoc

```diff
@@ -0,0 +1,75 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-creating-lvmcluster-using-rhacm_{context}"]
+= Creating an LVMCluster CR by using {rh-rhacm}
```

Comment:
@radeore same with the installation and uninstallation :) these can all be verified in a single run

Reply:
RHACM procs are being tracked in this PR - https://github.com/openshift/openshift-docs/pull/73279